2019-12-28 10:57:34,315 INFO: Microblog startup [in /mnt/c/Users/patel/Desktop/CPP_code/work/microblog/app/__init__.py:48]
2019-12-28 10:58:59,422 INFO: Microblog startup [in /mnt/c/Users/patel/Desktop/CPP_code/work/microblog/app/__init__.py:48]
2019-12-28 10:59:15,160 INFO: Microblog startup [in /mnt/c/Users/patel/Desktop/CPP_code/work/microblog/app/__init__.py:48]
2019-12-28 11:01:27,809 INFO: Microblog startup [in /mnt/c/Users/patel/Desktop/CPP_code/work/microblog/app/__init__.py:48]
2019-12-28 11:03:06,146 INFO: Microblog startup [in /mnt/c/Users/patel/Desktop/CPP_code/work/microblog/app/__init__.py:48]
2019-12-28 12:40:52,713 INFO: Microblog startup [in /mnt/c/Users/patel/Desktop/CPP_code/work/microblog/app/__init__.py:48]
2019-12-28 12:41:07,006 ERROR: Exception on /explore [GET] [in /mnt/c/Users/patel/Desktop/CPP_code/work/microblog/venv/lib/python3.6/site-packages/flask/app.py:1891]
Traceback (most recent call last):
  File "/mnt/c/Users/patel/Desktop/CPP_code/work/microblog/venv/lib/python3.6/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/mnt/c/Users/patel/Desktop/CPP_code/work/microblog/venv/lib/python3.6/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/mnt/c/Users/patel/Desktop/CPP_code/work/microblog/venv/lib/python3.6/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/mnt/c/Users/patel/Desktop/CPP_code/work/microblog/venv/lib/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/mnt/c/Users/patel/Desktop/CPP_code/work/microblog/venv/lib/python3.6/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/mnt/c/Users/patel/Desktop/CPP_code/work/microblog/venv/lib/python3.6/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/mnt/c/Users/patel/Desktop/CPP_code/work/microblog/venv/lib/python3.6/site-packages/flask_login/utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "/mnt/c/Users/patel/Desktop/CPP_code/work/microblog/app/routes.py", line 128, in explore
    return render_template("index.html", title='Home Page', form=form,
NameError: name 'form' is not defined
2019-12-28 12:41:49,813 ERROR: Exception on /explore [GET] [in /mnt/c/Users/patel/Desktop/CPP_code/work/microblog/venv/lib/python3.6/site-packages/flask/app.py:1891]
Traceback (most recent call last):
  File "/mnt/c/Users/patel/Desktop/CPP_code/work/microblog/venv/lib/python3.6/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/mnt/c/Users/patel/Desktop/CPP_code/work/microblog/venv/lib/python3.6/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/mnt/c/Users/patel/Desktop/CPP_code/work/microblog/venv/lib/python3.6/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/mnt/c/Users/patel/Desktop/CPP_code/work/microblog/venv/lib/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/mnt/c/Users/patel/Desktop/CPP_code/work/microblog/venv/lib/python3.6/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/mnt/c/Users/patel/Desktop/CPP_code/work/microblog/venv/lib/python3.6/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/mnt/c/Users/patel/Desktop/CPP_code/work/microblog/venv/lib/python3.6/site-packages/flask_login/utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "/mnt/c/Users/patel/Desktop/CPP_code/work/microblog/app/routes.py", line 128, in explore
    return render_template("index.html", title='Home Page', form=form,
NameError: name 'form' is not defined
2019-12-28 13:50:30,453 INFO: Microblog startup [in /mnt/c/Users/patel/Desktop/CPP_code/work/microblog/app/__init__.py:48]
2019-12-28 13:57:07,682 INFO: Microblog startup [in /mnt/c/Users/patel/Desktop/CPP_code/work/microblog/app/__init__.py:48]
2019-12-28 14:00:12,025 INFO: Microblog startup [in /mnt/c/Users/patel/Desktop/CPP_code/work/microblog/app/__init__.py:48]
2019-12-28 14:01:35,138 INFO: Microblog startup [in /mnt/c/Users/patel/Desktop/CPP_code/work/microblog/app/__init__.py:48]
2019-12-28 14:01:38,441 INFO: Microblog startup [in /mnt/c/Users/patel/Desktop/CPP_code/work/microblog/app/__init__.py:48]
2019-12-28 14:03:22,568 INFO: Microblog startup [in /mnt/c/Users/patel/Desktop/CPP_code/work/microblog/app/__init__.py:48]
2019-12-28 14:31:55,865 INFO: Microblog startup [in /mnt/c/Users/patel/Desktop/CPP_code/work/microblog/app/__init__.py:48]
2019-12-28 15:07:31,636 INFO: Microblog startup [in /mnt/c/Users/patel/Desktop/CPP_code/work/microblog/app/__init__.py:48]
2019-12-28 15:21:00,479 INFO: Microblog startup [in /mnt/c/Users/patel/Desktop/CPP_code/work/microblog/app/__init__.py:49]
2019-12-28 16:25:07,995 INFO: Microblog startup [in /mnt/c/Users/patel/Desktop/CPP_code/work/microblog/app/__init__.py:53]
2019-12-28 16:34:52,945 INFO: Microblog startup [in /mnt/c/Users/patel/Desktop/CPP_code/work/microblog/app/__init__.py:52]
